### PR TITLE
Fix ldap filter for Snom phones

### PR DESCRIPTION
**Pull request by andrea-marchionni**
Fix dap filter for Snom phones: Snom phones never add * for a search.

### CODE REVIEW

**Comment by andrea-marchionni** [comment]
nethesis/dev#6058

File: scripts/upgrade.d/012.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    'ldap_number_filter' => '(|(telephoneNumber=%*)(mobile=%*)(homePhone=%*))',
+    'ldap_name_filter' => '(|(cn=%*)(o=%*))'
+];
+
+foreach ($scopes as $model_id) {
+    $scope = new \Tancredi\Entity\Scope($model_id, $container['storage'], $container['logger']);
+    if(isset($scope->metadata['version']) && $scope->metadata['version'] >= 12) {
+        continue;
+    }
+    $scope->metadata['version'] = 12;
```
Comment (Stell0) [inline scripts/upgrade.d/012.php:54]:
```suggestion
    $scope->metadata['version'] = 7;
```

**Review by Stell0 — APPROVED** [review]

File: scripts/upgrade.d/012.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    'ldap_number_filter' => '(|(telephoneNumber=%*)(mobile=%*)(homePhone=%*))',
+    'ldap_name_filter' => '(|(cn=%*)(o=%*))'
+];
+
+foreach ($scopes as $model_id) {
+    $scope = new \Tancredi\Entity\Scope($model_id, $container['storage'], $container['logger']);
+    if(isset($scope->metadata['version']) && $scope->metadata['version'] >= 12) {
+        continue;
+    }
+    $scope->metadata['version'] = 7;
```
Comment (Stell0) [inline scripts/upgrade.d/012.php:54]:
```suggestion
    $scope->metadata['version'] = 12;
```

File: data/scopes/snom-D785.ini [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 scopeType = "model"
 displayName = "Snom D785"
-version = 6
+version = 7
```
Comment (Stell0) [inline data/scopes/snom-D785.ini:4]:
```suggestion
version = 12
```

File: data/scopes/snom-D765.ini [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 scopeType = "model"
 displayName = "Snom D765"
-version = 6
+version = 7
```
Comment (Stell0) [inline data/scopes/snom-D765.ini:4]:
```suggestion
version = 12
```

File: data/scopes/snom-D745.ini [hunk 5]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 scopeType = "model"
 displayName = "Snom D745"
-version = 6
+version = 7
```
Comment (Stell0) [inline data/scopes/snom-D745.ini:4]:
```suggestion
version = 12
```

File: data/scopes/snom-D735.ini [hunk 6]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 scopeType = "model"
 displayName = "Snom D735"
-version = 6
+version = 7
```
Comment (Stell0) [inline data/scopes/snom-D735.ini]:
```suggestion
version = 12
```

File: data/scopes/snom-D725.ini [hunk 7]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 scopeType = "model"
 displayName = "Snom D725"
-version = 6
+version = 7
```
Comment (Stell0) [inline data/scopes/snom-D725.ini]:
```suggestion
version = 12
```

File: data/scopes/snom-D712.ini [hunk 8]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 scopeType = "model"
 displayName = "Snom D712"
-version = 6
+version = 7
```
Comment (Stell0) [inline data/scopes/snom-D712.ini]:
```suggestion
version = 12
```

File: data/scopes/snom-D710.ini [hunk 9]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 scopeType = "model"
 displayName = "Snom D710"
-version = 6
+version = 7
```
Comment (Stell0) [inline data/scopes/snom-D710.ini]:
```suggestion
version = 12
```

File: data/scopes/snom-D385.ini [hunk 10]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 scopeType = "model"
 displayName = "Snom D385"
-version = 6
+version = 7
```
Comment (Stell0) [inline data/scopes/snom-D385.ini]:
```suggestion
version = 12
```

File: data/scopes/snom-D375.ini [hunk 11]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 scopeType = "model"
 displayName = "Snom D375"
-version = 6
+version = 7
```
Comment (Stell0) [inline data/scopes/snom-D375.ini]:
```suggestion
version = 12
```

File: data/scopes/snom-D315.ini [hunk 12]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 scopeType = "model"
 displayName = "Snom D315"
-version = 6
+version = 7
```
Comment (Stell0) [inline data/scopes/snom-D315.ini]:
```suggestion
version = 12
```

File: data/scopes/snom-D305.ini [hunk 13]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 scopeType = "model"
 displayName = "Snom D305"
-version = 6
+version = 7
```
Comment (Stell0) [inline data/scopes/snom-D305.ini]:
```suggestion
version = 12
```

File: data/scopes/snom-D120.ini [hunk 14]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 scopeType = "model"
 displayName = "Snom D120"
-version = 6
+version = 7
```
Comment (Stell0) [inline data/scopes/snom-D120.ini]:
```suggestion
version = 12
```

**Review by Stell0 — APPROVED** [review]

**Pull request merged**